.github/workflows/test.yml

```diff
@@ -68,3 +68,34 @@ jobs:
 
       - name: Test
         run: ctest -VV --test-dir build/tests/cpp --parallel 2
+
+  test-python-from-cmake:
+  # To support hybrid C++ applications, the python package may also be
+  # generated using the CMake packaging, in addition and simultaneously
+  # to the C++ package. This tests that.
+    name: "Test-Python-From-CMake"
+    runs-on: ubuntu-latest
+    container:
+      image: aswf/ci-vfxall:2022-clang14.3
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install Traitgen
+        run: python -m pip install openassetio-traitgen==1.0.0a6
+
+      - name: Configure CMake build
+        run: >
+          cmake -S . -B build -G Ninja
+          -DOPENASSETIO_MEDIACREATION_GENERATE_PYTHON=ON
+          -DOPENASSETIO_MEDIACREATION_PYTHON_SITEDIR="hybridpython"
```

Comment:
I actually had that originally, but through that testing our custom variable was more important the re-testing the logic of the default.

Could have both, but I think we'd have do do a whole other case which seems a bit ... much? Will do if you think it's important, ortherwise i'll just leave as is.

Reply:
I agree testing both cases seems a bit much, I was just wondering which was the more important case to have an automated test for.  

Our default behaviour of finding Python and constructing the site-packages path seems more complex and more likely to break.  But then even if we did test the default behaviour, it would only be for Linux.

I still lean slightly toward testing the default behaviour, but its not too important, lets leave as is.